Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,14 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
```

Comment:
Please renaming `str` to `string`, no abbreviations in WebKit.

Reply:
Fixed!